recipe/meta.yaml

```diff
@@ -23,39 +27,45 @@ build:
 requirements:
   build:
     - {{ compiler('c') }}
-    - patch               # [not win]
-    - m2-patch            # [win]
   host:
     - python
     - pip
     - setuptools
     - wheel
-    - cython
-    - proj
+    - cython >=0.28.4
+    - proj >={{ proj_min }}
   run:
     - python
-    - proj
+    - proj >={{ proj_min }}
     - certifi
 
 test:
+  # NOTE to future maintainers: Reenable full testing when pandas is available for python 3.10.
   source_files:
     - test
   requires:
     - pytest
-    - shapely
+    - shapely  # [(not osx or py<=39) and (not s390x)]
     - numpy
-    - pandas
-    - xarray
+    - pandas  # [py<310 or osx]  Not available for python 3.10 right now for win or lin 64
+    - xarray  # [py<310 or osx] Depends on pandas
   imports:
     - pyproj
   commands:
     - pyproj --verbose
```

Comment:
It would be nice to add the command `- pyproj --help`

Reply:
Added!